packages/react-data-grid/src/RowGroup.js

```diff
@@ -95,7 +95,7 @@ const  DefaultRowGroupRenderer = (props) => {
   return (
     <div style={style} onKeyDown={onKeyDown} tabIndex={0}>
       <span className="row-expand-icon" style={{float: 'left', marginLeft: marginLeft, cursor: 'pointer'}} onClick={props.onRowExpandClick} >{props.isExpanded ? String.fromCharCode('9660') : String.fromCharCode('9658')}</span>
-      <strong>{props.columnGroupName} : {props.name}</strong>
```

Comment:
My main concern would be breaking this for people that are already using it, because they are expecting the previous and we will change it in our core and break their functionality